packages/theme-chalk/src/select.scss

```
@@ -205,12 +205,16 @@
 
   @include e(input-wrapper) {
     flex: 1;
+    // Allow the wrapper to shrink below the input's intrinsic width so an
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)